README.md

```diff
@@ -17,7 +17,7 @@ Fedora Remix for WSL is provided on a community-support basis. There is no suppo
 ### General WSL and Fedora documentation:
 
 - [Install Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
-- [Fedora 29 User Documentation](https://docs.fedoraproject.org/en-US/fedora/f29/)
+- [Fedora 30 User Documentation](https://docs.fedoraproject.org/en-US/fedora/f30/)
```

Comment:
Update to F31

Reply:
We are working on this upgrade. Thanks for bring it to the table